tests/test_rabfe_topology.py

```diff
@@ -107,10 +107,8 @@ def test_base_topology_standard_decoupling():
 def test_dual_topology_standard_decoupling():
 
     # this class is used in double decoupling stages of the RABFE protocol. It modifies the
-    # DualTopology class in two ways:
-    # 1) the torsions between non-ring atoms are turned off
```

Comment:
This seemed wrong at this point, I hope that I am not mistaken

Reply:
Double-checking: there is a test specific to ring torsions https://github.com/proteneer/timemachine/blob/4e4b861002a018b822d1311e54fc3a2e2be4e9d1/tests/test_rabfe_topology.py#L15-L43 , but I'll need to dig a bit more to confirm

Reply:
Correct, since https://github.com/proteneer/timemachine/pull/473